lib/PPI/Statement/Include.pm

```diff
@@ -236,6 +238,28 @@ sub arguments {
 	return @args;
 }
 
+sub feature_mods {
+	my ($self) = @_;
+
+	return if $self->type eq "require";
+
+	if ( my $perl_version = $self->version ) {
+		## crude proof of concept hack due to above
+		return { signatures => 1 } if $perl_version >= 5.035;
```

Comment:
This seems to be the huge TBD. How do we know if signatures are enabled? I continue to ask: why do we need to know this? if you see a signature, then why not say "yep it's a signature" and hook in the tokens?